pages/Configuring/Window-Rules.md

```diff
@@ -316,7 +314,7 @@ where `rule` is the rule and `namespace` is the namespace regex (find
 namespaces in `hyprctl layers`) or `address` is an address in the form of
 `address:0x[hex]`.
 
-### Rules
+### List of Rules
```

Comment:
This will break existing links. I'd rather leave headings as they are.

Reply:
the linter complained that there is already a heading named "rules", i couldnt find anything that linked here, just at ##rules in line 95. but i can revert that change and just commit the others, if thats ok?

Reply:
Yeah, the rest are ok.

Reply:
Newest commit should be good to go :)
can we merge it like that or do i have to open another PR - havent contributed much in public projects up until now, sorry if thats a newb question.

Reply:
Yeah, it can be merged. No need for another PR.